data/campaigns/Two_Brothers/scenarios/01_Rooting_Out_a_Mage.cfg

```diff
@@ -47,32 +47,40 @@
     [story]
         [part]
             music=revelation.ogg
-            story=_ "The remote freehold of Maghre in the western reaches of the Kingdom of Wesnoth was once a peaceful place, its inhabitants largely unaware of the comings and goings of the wider world. Wars and the rumor of wars touched them not, until the day a dark mage settled in the region and began seeking sacrifices for his evil summonings."
-            background=story/Two_Brothers_M1P1.webp
+            # po: This text is shown with the journey map, showing the village in the foothills along the route between Aldril and Dan’Tonk.
+            # po: I'm assuming the beacons either use smoke signals or basic fire signals, with a few agreed ones to request goods or indicate that surplus is for sale.
+            story=_ "The remote freehold of Maghre in the western reaches of the Kingdom of Wesnoth was once a peaceful place, its inhabitants largely unaware of the comings and goings of the wider world. Wars and the rumor of wars touched them not.
+
+Even merchants were rarely seen. Caravans between Aldril and Dan’Tonk would sometimes send a wagon to the village when called by the beacon tower."
```

Comment:
Probably should just be one paragraph, no need to separate.

Reply:
I'll make the change, but when combining it into a single paragraph, I also had to make the two sentences into one.